README.markdown

```diff
@@ -57,7 +57,7 @@ localhost and a production server. :
         "local": {
             "host": "http://localhost:8000/rest_api/v1/",
             "develop": true,
-            "insecure": false
+            "ssl-verify": false
```

Comment:
Yeah. As we prefer ssl-verify, we should update this.
In our internal repo, the file pdc_client_config.json should also be updated. But that should be in another patch